modules/clightning.nix

```diff
@@ -111,6 +110,10 @@ in {
         while [[ ! -e ${cfg.dataDir}/bitcoin/lightning-rpc ]]; do
             sleep 0.1
         done
+        # give group "search" access to allow clightning group processes to find lightning-rpc
+        if [[ -e ${cfg.dataDir}/bitcoin ]]; then
+          chmod g+X ${cfg.dataDir}/bitcoin
+        fi
       '';
     };
   };
```

Comment:
I think we can remove group read access from the config file (chmodded in preStart). Otherwise spark-wallet, lightning-charge, etc. can read the bitcoin rpc credentials. This was probably necessary back when the operator did not sudo to the clightning user to run lightning-cli.

Reply:
Resolved in a0e579d